src/test/java/itm/oss/splitter/ExporterTest.java

```diff
@@ -0,0 +1,55 @@
+package itm.oss.splitter;
+
+import org.junit.jupiter.api.Test;
+import java.io.File;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import static org.junit.jupiter.api.Assertions.*;
+import java.math.*;
+
+public class ExporterTest {
+
+    @Test
+    public void testWritePaymentsCsv_createsFileWithHeader() throws Exception {
+        
+        ArrayList<Payment> pays = new ArrayList<>();
+        pays.add(new Payment("Alice", "Bob",  new BigDecimal("30.0")));
+        
+        pays.add(new Payment("Charlie", "Alice",  new BigDecimal("33.3333333")));
+
+        pays.add(new Payment("Hong", "Lee",  new BigDecimal("2500.0")));
+
+        pays.add(new Payment(null, null, null));
+
+        String path = "data/expense.sample.csv";
+
+        Exporter.writePaymentsCsv(path, pays);
+
+        File file = new File(path);
+        assertTrue(file.exists(), "csv file needs to be created");
+
+        String content = Files.readString(file.toPath());
+        assertTrue(content.contains("from,to,amount"), "needs to include csv header");
+        assertTrue(content.contains("Alice,Bob,30"), "needs to include payment record");
+    }
+
+    @Test
+    public void testWritePaymentsCsv_handlesEmptyList() throws Exception {
+        
```

Comment:
To further strengthen the reliability and business correctness of this feature, how about adding one
more test that explicitly covers zero and negative amounts?

Examples of cases worth verifying:
- Negative amount → whether it is allowed and how it is represented in the CSV. 

```suggestion
@Test
void writePaymentsCsv_HandlesNegativeAmounts() throws Exception {
    Path csvPath = Paths.get("data", "edgecases", "negative.csv");
    Files.createDirectories(csvPath.getParent());
    Files.deleteIfExists(csvPath);

    List<Payment> payments = List.of(
        new Payment("Bob", "Charlie", new BigDecimal("-15.5"))
    );

    Exporter.writePaymentsCsv(csvPath.toString(), payments);

    List<String> lines = Files.readAllLines(csvPath);
    assertEquals(2, lines.size()); // header + 1 row
    assertEquals("from,to,amount", lines.get(0));
    assertEquals("Bob,Charlie,-15.50", lines.get(1)); // expecting two decimal places
}


```

Reply:
Applied negative case code, creating edgecases file in data folder. Thank you for your review!!

Reply:
Added a new test method to verify handling of negative amounts as suggested.
Thanks for the helpful review!

Reply:
Great, thanks for the quick update. The new edge cases make the test coverage more solid.